source/installation_guide/index.rst

```diff
@@ -12,3 +12,28 @@ Installation guide
    dovecot_community_repositories/index
    upgrading/index
    startup_scripts/index
+
+Target Platform
+^^^^^^^^^^^^^^^
+
+As part of improving maintainability and sustainability of
+:ref:`ox_dovecot_pro` this project defines a target platform specification and
+a minimum language standard beginning with version v3.0.0. As a platform
```

Comment:
Hmm. I think better say v2.4.0 here, since that's the community version, and anybody compiling Dovecot will be using v2.4 rather than v3.0.

Also I don't think these changes are necessarily even related to OX Dovecot Pro, so the sentence could instead just say "... sustainability of the code, this project ..."